src/client/components/equalizer/streamDetect/streamDetect.css

```diff
@@ -0,0 +1,3 @@
+#stream_detecting {
+  margin-left: 10px;
+}
```

Comment:
missed empty line at the end of the file.

Reply:
done)